frontend/lib/main.dart

```diff
@@ -45,7 +45,7 @@ class MyApp extends StatelessWidget {
           theme: ThemeData(
             primarySwatch: Colors.blue,
           ),
-          home: const MainScreen(),
+          home: const WelcomeScreen(),
```

Comment:
For now we should keep MainScreen as home screen, I'm working on fix for this

Reply:
Undo change